tools/buildsys/src/cache.rs

```diff
@@ -65,9 +67,7 @@ impl LookasideCache {
             }
 
             // next check with upstream, if permitted
-            if f.force_upstream.unwrap_or(false)
-                || std::env::var("BUILDSYS_UPSTREAM_SOURCE_FALLBACK") == Ok("true".to_string())
-            {
+            if f.force_upstream.unwrap_or(false) || upstream_fallback {
```

Comment:
Will this be the same as 

```suggestion
            if f.force_upstream.unwrap_or(upstream_fallback) {
```

Reply:
True but it seems harder to reason about to me.